### DISCUSSION TITLE
Neurons identified differently between Caiman and CNMF_E

### DISCUSSION THREAD

**Discussion by wsshen** [flatironinstitute/CaImAn#1382] [General]
Has anyone tested the differences between Caiman and the CNMF-E, https://github.com/zhoupc/CNMF_E? I find that even with the same min_corr and min_pnr, CNMF-E would find more neurons than Caiman, and CNMF-E identified neurons whose corr and pnr were much lower than min_corr and min_pnr, whereas Caiman obeyed the min_corr and min_pnr. It's also acknowledged that "However, these two implementations are not exactly the same in some details and we are still working to make them consistent. ". What are the causes for these differences?

**Comment by pgunn** [comment]
We talked about this a bit on gitter; wsshen found out that CNMF-E does filtering differently in its second step (it does a 2d gaussian for that step rather than a disk filter for both as Caiman does).

It's unclear which method is better for most datasets; both seem viable filtering methods.
